clio-server/src/main/scala/org/broadinstitute/clio/server/service/PersistenceService.scala

```diff
@@ -48,7 +47,7 @@ class PersistenceService private (persistenceDAO: PersistenceDAO, searchDAO: Sea
 
 object PersistenceService {
 
-  def apply(app: ClioApp): PersistenceService = {
-    new PersistenceService(app.persistenceDAO, app.searchDAO)
+  def apply(persistenceDAO: PersistenceDAO, searchDAO: SearchDAO): PersistenceService = {
```

Comment:
I'd just delete this apply method, it's not buying us anything

Reply:
Just a note: scrapping these requires making each Service class private[server] rather than private or private[service] (to access from ClioServer). Seems like an acceptable change to me.